docs/premium-features/break-the-monolith.md

```diff
@@ -6,61 +6,64 @@ slug: /break-the-monolith
+
+This guide will show you how to effectively use this feature for your services.
 
 :::note
-The "Break The Monolith" feature is available exclusively to Enterprise workspaces, with trial workspaces having limited access.
+The "Break The Monolith" feature is **available exclusively to Enterprise workspaces including trial workspaces**.
 :::
 
 ## What is "Break The Monolith"?
 
-"Breaking the Monolith" is designed to help you transition a legacy app with a monolithic structure to a microservice architecture.
+"Break The Monolith" helps you transition any monolithic service, including monolithic legacy systems, to a modular microservice architecture. It allows for both manual and AI-driven restructuring of services.
 
-This feature consists of two parts:
+"Breaking the Monolith" is designed to help you transition any app with a monolithic structure to a microservice architecture.
 
-1. **AI-powered Entity Analysis**: We assess your service's entities to suggest optimal division into microservices.
-2. **Service Migration**: Based on that analysis, we migrate entities across services.
+This feature focuses on the following 3 parts:
 
-:::note
-**This feature uses LLMs to analyze and suggest ideal microservices structures**. It requires sharing details of your entities and fields with LLMs for analysis. We ensure the highest privacy standards and use your data only to improve your project's architecture.
-:::
+1. **Manual Service Redesign**: Manually move entities between existing or new services to create tailored micro-services.
+2. **AI-powered Entity Analysis and Service Suggestions**: As an optional feature, AI analyzes and proposes the most optimal way to distribute your entities among services, offering smart recommendations for service composition and entity organization.
+3. **Service and Database Creation**: Based on the new architecture, Amplication creates new services and updates databases to support service separation.
 
 ## How To Use "Break The Monolith"
 
 ### Step 1: Access the Architecture Tab
 
-A new "Architecture" tab is now part of every project interface.
-
-It features two modes: **Overview** and **Redesign**.
+Visit the new "Architecture" tab in your project interface, featuring two modes: **Overview** and **Redesign**.
 
 #### Overview Mode
 
 Overview mode is the view-only default mode for viewing your services and entities.
 
 #### Redesign Mode
 
-Redesign mode allows you make modifications to your service. It lets you rename your service, update descriptions, and most importantly, migrate entities.
+Redesign mode allows you redesign one service at a time and migrate entities.
 
-To migrate entities to other services you can drag and drop them into the appropriate service. After making changes, you can click on the **Apply** button to implement those changes.
+### Step 2: Manual Service Redesign
+
+After entering Redesign Mode, you can manually move your entity between existing or new services to create miro-services.
```

Comment:
Maybe we need also to add that one can create new service/s as part of the redesign mode.

Reply:
In addition- shouldn't it be "move your entities" in plural?